submodules/client.js

```diff
@@ -57,6 +57,22 @@ class OpenNodeClient {
     }
   }
 
+  async initiateExchange(exchange) {
+   try {
+     let new_instance = axios.create();
+     new_instance.defaults.baseURL = (this.env === 'live') ? 'https://api.opennode.com/v2' : 'https://dev-api.opennode.com/v2';
+     new_instance.defaults.timeout = 15000;
+     new_instance.defaults.headers = { 'Authorization': this.api_key, 'user_agent': version };
+
+     const response = await new_instance.post('/exchanges', exchange);
```

Comment:
Is there a reason to create a new axios instance over `this.instance`?

Reply:
Ah, nevermind, just realized this is for `v2` and `this.instance` is for `v1`

Reply:
hey @johnnyasantoss - 

I believe "this.instance" refers to the v1 api URL whereas the initiateExchange ep is only on v2. 

Similar to how "initiateWithdrawalAsync"  is setup 

"async initiateWithdrawalAsync(withdrawal) {
    try {
      let new_instance = axios.create();
      new_instance.defaults.baseURL = (this.env === 'live') ? 'https://api.opennode.com/v2' : 'https://dev-api.opennode.com/v2';
      new_instance.defaults.timeout = 15000;
      new_instance.defaults.headers = { 'Authorization': this.api_key, 'user_agent': version };

      const response = await new_instance.post('/withdrawals', withdrawal);
      return response.data.data;
    }
    catch (error) {
      throw Exception(error.response.status, error.response.statusText, error.response.data.message);
    }
  }"